### DISCUSSION TITLE
Student developer pack benefits

### DISCUSSION THREAD

**Discussion by goblegue** [community/community#127642] [GitHub Education]
### Hi everyone,

**Hello**
I have been approved for student developer pack but still have not received the benefits and it has been 4 days since I have been approved

**Comment by davevad93** [comment]
Hi @goblegue ,

Be patient because the waiting time may vary from a few days to a few weeks. When your benefits are available you'll see the following message on the application page:

-  `Congrats! Your request for @goblegue was approved and your benefits are now available.`.
